dotnet/samples/Concepts/Agents/DeclarativeAgents.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using System.Text;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Agents;
+using Microsoft.SemanticKernel.Agents.Chat;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+using Plugins;
+using Resources;
+using static Agents.ChatCompletion_FunctionTermination;
+
+namespace Agents;
+
+public class DeclarativeAgents(ITestOutputHelper output) : BaseAgentsTest(output)
+{
+    [InlineData("SchedulingAssistant.json", "Read the body of my last five emails, if any contain a meeting request for today, check that it's already on my calendar, if not, call out which email it is.")]
+    [Theory]
+    public async Task LoadsAgentFromDeclarativeAgentManifestAsync(string agentFileName, string input)
+    {
+        var kernel = CreateKernelWithFilter();
+        var manifestLookupDirectory = Path.Combine(Directory.GetCurrentDirectory(), "..", "..", "..", "Resources", "DeclarativeAgents");
+        var manifestFilePath = Path.Combine(manifestLookupDirectory, agentFileName);
+
+        var parameters = await CopilotAgentBasedPlugins.GetAuthenticationParametersAsync();
+
+        var agent = await kernel.CreateChatCompletionAgentFromDeclarativeAgentManifestAsync<ChatCompletionAgent>(manifestFilePath, parameters);
+
+        Assert.NotNull(agent);
+        Assert.NotNull(agent.Name);
+        Assert.NotEmpty(agent.Name);
+        Assert.NotNull(agent.Description);
+        Assert.NotEmpty(agent.Description);
+        Assert.NotNull(agent.Instructions);
+        Assert.NotEmpty(agent.Instructions);
+
+        ChatMessageContent message = new(AuthorRole.User, input);
+        ChatHistory chatHistory = [message];
+        StringBuilder sb = new();
+        await foreach (ChatMessageContent response in agent.InvokeAsync(chatHistory))
```

Comment:
One question I had along this topic is why is semantic kernel sending the expected schema back to the LLM? (as a property on the content property of ToolChatMessage)

Reply:
> @SergeyMenshykh here I'm struggling a bit with the test design. I keep running into the following exception `This model's maximum context length is 8192 tokens. However, your messages resulted in 96877 tokens (95278 in the messages, 1599 in the functions). Please reduce the length of the messages or functions.`
> 
> I think what's happening if you look at the prompt and the functions being loaded from plugins during execution:
> 
> 1. first call to get messages, everything is fine.
> 2. second call to get calendar events, the result from the first call is appended to the history, but the JSON payload.
> 
> But I'm not sure that's the issue at the moment. I'd like to know how I could confirm that, and pointers on how to better design this test please.

Hi @baywet, looking at the error message, it seems the test sent more input tokens to the model than the model can handle. If that is the case, I would recommend doing the following to identify the root cause:   
  
1. Look at the chat history and try to identify what could have consumed 96877 tokens - chat message content, function call content, or function result content.  
2. Look at the list of functions advertised to the model and see if all of them are relevant to the scenario.  
   
The solution will depend on the root cause and might require tweaking the prompt, reducing the number of functions, reducing chat history as described in the [Managing Chat History for Large Language Models (LLMs)](https://devblogs.microsoft.com/semantic-kernel/managing-chat-history-for-large-language-models-llms/) blog post, or fixing the code if it is caused by a bug.

CC: @crickman @markwallace-microsoft

Reply:
> One question I had along this topic is why is semantic kernel sending the expected schema back to the LLM? (as a property on the content property of ToolChatMessage)

Could you please provide a link to the line where it happens and maybe a snapshot of the content that is supplied to the property?

Reply:
this is what I'm referring to https://github.com/microsoft/semantic-kernel/blob/560e4c96cdcb0c2869ba451dc34379705f738420/dotnet/src/SemanticKernel.Abstractions/Functions/RestApiOperationResponse.cs#L27

Which effectively results in the following tool message (content removed for brievety)

```json
{
	"Content": "(removed)",
	"ContentType": "application/json; odata.metadata=minimal; odata.streaming=true; IEEE754Compatible=false; charset=utf-8",
	"ExpectedSchema": {
		"title": "Collection of message",
		"type": "object",
		"allOf": [
			{
				"title": "Base collection pagination and count responses",
				"type": "object",
				"properties": {
					"@odata.nextLink": {
						"type": "string",
						"nullable": true
					}
				}
			},
			{
				"type": "object",
				"properties": {
					"value": {
						"type": "array",
						"items": {
							"allOf": [
								{
									"allOf": [
										{
											"title": "entity",
											"type": "object",
											"properties": {
												"id": {
													"type": "string",
													"description": "The unique identifier for an entity. Read-only."
												}
											}
										},
										{
											"title": "outlookItem",
											"type": "object",
											"properties": {
												"categories": {
													"type": "array",
													"items": {
														"type": "string",
														"nullable": true
													},
													"description": "The categories associated with the item"
												},
												"changeKey": {
													"type": "string",
													"description": "Identifies the version of the item. Every time the item is changed, changeKey changes as well. This allows Exchange to apply changes to the correct version of the object. Read-only.",
													"nullable": true
												},
												"createdDateTime": {
													"pattern": "^[0-9]{4,}-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T([01][0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9]([.][0-9]{1,12})?(Z|[\u002B-][0-9][0-9]:[0-9][0-9])$",
													"type": "string",
													"description": "The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is 2014-01-01T00:00:00Z",
													"format": "date-time",
													"nullable": true
												},
												"lastModifiedDateTime": {
													"pattern": "^[0-9]{4,}-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T([01][0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9]([.][0-9]{1,12})?(Z|[\u002B-][0-9][0-9]:[0-9][0-9])$",
													"type": "string",
													"description": "The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is 2014-01-01T00:00:00Z",
													"format": "date-time",
													"nullable": true
												}
											}
										}
									]
								},
								{
									"title": "message",
									"type": "object",
									"properties": {
										"bccRecipients": {
											"type": "array",
											"items": {
												"title": "recipient",
												"type": "object",
												"properties": {
													"emailAddress": {
														"anyOf": [
															{
																"title": "emailAddress",
																"type": "object",
																"properties": {
																	"address": {
																		"type": "string",
																		"description": "The email address of the person or entity.",
																		"nullable": true
																	},
																	"name": {
																		"type": "string",
																		"description": "The display name of the person or entity.",
																		"nullable": true
																	}
																}
															},
															{
																"type": "object",
																"nullable": true
															}
														],
														"description": "The recipient\u0027s email address."
													}
												}
											},
											"description": "The Bcc: recipients for the message."
										},
										"body": {
											"anyOf": [
												{
													"title": "itemBody",
													"type": "object",
													"properties": {
														"content": {
															"type": "string",
															"description": "The content of the item.",
															"nullable": true
														},
														"contentType": {
															"anyOf": [
																{
																	"title": "bodyType",
																	"enum": [
																		"text",
																		"html"
																	],
																	"type": "string"
																},
																{
																	"type": "object",
																	"nullable": true
																}
															],
															"description": "The type of the content. Possible values are text and html."
														}
													}
												},
												{
													"type": "object",
													"nullable": true
												}
											],
											"description": "The body of the message. It can be in HTML or text format. Find out about safe HTML in a message body."
										},
										"bodyPreview": {
											"type": "string",
											"description": "The first 255 characters of the message body. It is in text format.",
											"nullable": true
										},
										"ccRecipients": {
											"type": "array",
											"items": {
												"title": "recipient",
												"type": "object",
												"properties": {
													"emailAddress": {
														"anyOf": [
															{
																"title": "emailAddress",
																"type": "object",
																"properties": {
																	"address": {
																		"type": "string",
																		"description": "The email address of the person or entity.",
																		"nullable": true
																	},
																	"name": {
																		"type": "string",
																		"description": "The display name of the person or entity.",
																		"nullable": true
																	}
																}
															},
															{
																"type": "object",
																"nullable": true
															}
														],
														"description": "The recipient\u0027s email address."
													}
												}
											},
											"description": "The Cc: recipients for the message."
										},
										"conversationId": {
											"type": "string",
											"description": "The ID of the conversation the email belongs to.",
											"nullable": true
										},
										"conversationIndex": {
											"type": "string",
											"description": "Indicates the position of the message within the conversation.",
											"format": "base64url",
											"nullable": true
										},
										"flag": {
											"anyOf": [
												{
													"title": "followupFlag",
													"type": "object",
													"properties": {
														"completedDateTime": {
															"anyOf": [
																{
																	"title": "dateTimeTimeZone",
																	"type": "object",
																	"properties": {
																		"dateTime": {
																			"type": "string",
																			"description": "A single point of time in a combined date and time representation ({date}T{time}; for example, 2017-08-29T04:00:00.0000000)."
																		},
																		"timeZone": {
																			"type": "string",
																			"description": "Represents a time zone, for example, \u0027Pacific Standard Time\u0027. See below for more possible values.",
																			"nullable": true
																		}
																	}
																},
																{
																	"type": "object",
																	"nullable": true
																}
															],
															"description": "The date and time that the follow-up was finished."
														},
														"dueDateTime": {
															"anyOf": [
																{
																	"title": "dateTimeTimeZone",
																	"type": "object",
																	"properties": {
																		"dateTime": {
																			"type": "string",
																			"description": "A single point of time in a combined date and time representation ({date}T{time}; for example, 2017-08-29T04:00:00.0000000)."
																		},
																		"timeZone": {
																			"type": "string",
																			"description": "Represents a time zone, for example, \u0027Pacific Standard Time\u0027. See below for more possible values.",
																			"nullable": true
																		}
																	}
																},
																{
																	"type": "object",
																	"nullable": true
																}
															],
															"description": "The date and time that the follow-up is to be finished. Note: To set the due date, you must also specify the startDateTime; otherwise, you get a 400 Bad Request response."
														},
														"flagStatus": {
															"anyOf": [
																{
																	"title": "followupFlagStatus",
																	"enum": [
																		"notFlagged",
																		"complete",
																		"flagged"
																	],
																	"type": "string"
																},
																{
																	"type": "object",
																	"nullable": true
																}
															],
															"description": "The status for follow-up for an item. Possible values are notFlagged, complete, and flagged."
														},
														"startDateTime": {
															"anyOf": [
																{
																	"title": "dateTimeTimeZone",
																	"type": "object",
																	"properties": {
																		"dateTime": {
																			"type": "string",
																			"description": "A single point of time in a combined date and time representation ({date}T{time}; for example, 2017-08-29T04:00:00.0000000)."
																		},
																		"timeZone": {
																			"type": "string",
																			"description": "Represents a time zone, for example, \u0027Pacific Standard Time\u0027. See below for more possible values.",
																			"nullable": true
																		}
																	}
																},
																{
																	"type": "object",
																	"nullable": true
																}
															],
															"description": "The date and time that the follow-up is to begin."
														}
													}
												},
												{
													"type": "object",
													"nullable": true
												}
											],
											"description": "The flag value that indicates the status, start date, due date, or completion date for the message."
										},
										"from": {
											"anyOf": [
												{
													"title": "recipient",
													"type": "object",
													"properties": {
														"emailAddress": {
															"anyOf": [
																{
																	"title": "emailAddress",
																	"type": "object",
																	"properties": {
																		"address": {
																			"type": "string",
																			"description": "The email address of the person or entity.",
																			"nullable": true
																		},
																		"name": {
																			"type": "string",
																			"description": "The display name of the person or entity.",
																			"nullable": true
																		}
																	}
																},
																{
																	"type": "object",
																	"nullable": true
																}
															],
															"description": "The recipient\u0027s email address."
														}
													}
												},
												{
													"type": "object",
													"nullable": true
												}
											],
											"description": "The owner of the mailbox from which the message is sent. In most cases, this value is the same as the sender property, except for sharing or delegation scenarios. The value must correspond to the actual mailbox used. Find out more about setting the from and sender properties of a message."
										},
										"hasAttachments": {
											"type": "boolean",
											"description": "Indicates whether the message has attachments. This property doesn\u0027t include inline attachments, so if a message contains only inline attachments, this property is false. To verify the existence of inline attachments, parse the body property to look for a src attribute, such as \u003CIMG src=\u0027cid:image001.jpg@01D26CD8.6C05F070\u0027\u003E.",
											"nullable": true
										},
										"importance": {
											"anyOf": [
												{
													"title": "importance",
													"enum": [
														"low",
														"normal",
														"high"
													],
													"type": "string"
												},
												{
													"type": "object",
													"nullable": true
												}
											],
											"description": "The importance of the message. The possible values are: low, normal, and high."
										},
										"inferenceClassification": {
											"anyOf": [
												{
													"title": "inferenceClassificationType",
													"enum": [
														"focused",
														"other"
													],
													"type": "string"
												},
												{
													"type": "object",
													"nullable": true
												}
											],
											"description": "The classification of the message for the user, based on inferred relevance or importance, or on an explicit override. The possible values are: focused or other."
										},
										"internetMessageHeaders": {
											"type": "array",
											"items": {
												"title": "internetMessageHeader",
												"type": "object",
												"properties": {
													"name": {
														"type": "string",
														"description": "Represents the key in a key-value pair.",
														"nullable": true
													},
													"value": {
														"type": "string",
														"description": "The value in a key-value pair.",
														"nullable": true
													}
												}
											},
											"description": "A collection of message headers defined by RFC5322. The set includes message headers indicating the network path taken by a message from the sender to the recipient. It can also contain custom message headers that hold app data for the message.  Returned only on applying a $select query option. Read-only."
										},
										"internetMessageId": {
											"type": "string",
											"description": "The message ID in the format specified by RFC2822.",
											"nullable": true
										},
										"isDeliveryReceiptRequested": {
											"type": "boolean",
											"description": "Indicates whether a read receipt is requested for the message.",
											"nullable": true
										},
										"isDraft": {
											"type": "boolean",
											"description": "Indicates whether the message is a draft. A message is a draft if it hasn\u0027t been sent yet.",
											"nullable": true
										},
										"isRead": {
											"type": "boolean",
											"description": "Indicates whether the message has been read.",
											"nullable": true
										},
										"isReadReceiptRequested": {
											"type": "boolean",
											"description": "Indicates whether a read receipt is requested for the message.",
											"nullable": true
										},
										"parentFolderId": {
											"type": "string",
											"description": "The unique identifier for the message\u0027s parent mailFolder.",
											"nullable": true
										},
										"receivedDateTime": {
											"pattern": "^[0-9]{4,}-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T([01][0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9]([.][0-9]{1,12})?(Z|[\u002B-][0-9][0-9]:[0-9][0-9])$",
											"type": "string",
											"description": "The date and time the message was received.  The date and time information uses ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is 2014-01-01T00:00:00Z.",
											"format": "date-time",
											"nullable": true
										},
										"replyTo": {
											"type": "array",
											"items": {
												"title": "recipient",
												"type": "object",
												"properties": {
													"emailAddress": {
														"anyOf": [
															{
																"title": "emailAddress",
																"type": "object",
																"properties": {
																	"address": {
																		"type": "string",
																		"description": "The email address of the person or entity.",
																		"nullable": true
																	},
																	"name": {
																		"type": "string",
																		"description": "The display name of the person or entity.",
																		"nullable": true
																	}
																}
															},
															{
																"type": "object",
																"nullable": true
															}
														],
														"description": "The recipient\u0027s email address."
													}
												}
											},
											"description": "The email addresses to use when replying."
										},
										"sender": {
											"anyOf": [
												{
													"title": "recipient",
													"type": "object",
													"properties": {
														"emailAddress": {
															"anyOf": [
																{
																	"title": "emailAddress",
																	"type": "object",
																	"properties": {
																		"address": {
																			"type": "string",
																			"description": "The email address of the person or entity.",
																			"nullable": true
																		},
																		"name": {
																			"type": "string",
																			"description": "The display name of the person or entity.",
																			"nullable": true
																		}
																	}
																},
																{
																	"type": "object",
																	"nullable": true
																}
															],
															"description": "The recipient\u0027s email address."
														}
													}
												},
												{
													"type": "object",
													"nullable": true
												}
											],
											"description": "The account that is actually used to generate the message. In most cases, this value is the same as the from property. You can set this property to a different value when sending a message from a shared mailbox, for a shared calendar, or as a delegate. In any case, the value must correspond to the actual mailbox used. Find out more about setting the from and sender properties of a message."
										},
										"sentDateTime": {
											"pattern": "^[0-9]{4,}-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T([01][0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9]([.][0-9]{1,12})?(Z|[\u002B-][0-9][0-9]:[0-9][0-9])$",
											"type": "string",
											"description": "The date and time the message was sent.  The date and time information uses ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is 2014-01-01T00:00:00Z.",
											"format": "date-time",
											"nullable": true
										},
										"subject": {
											"type": "string",
											"description": "The subject of the message.",
											"nullable": true
										},
										"toRecipients": {
											"type": "array",
											"items": {
												"title": "recipient",
												"type": "object",
												"properties": {
													"emailAddress": {
														"anyOf": [
															{
																"title": "emailAddress",
																"type": "object",
																"properties": {
																	"address": {
																		"type": "string",
																		"description": "The email address of the person or entity.",
																		"nullable": true
																	},
																	"name": {
																		"type": "string",
																		"description": "The display name of the person or entity.",
																		"nullable": true
																	}
																}
															},
															{
																"type": "object",
																"nullable": true
															}
														],
														"description": "The recipient\u0027s email address."
													}
												}
											},
											"description": "The To: recipients for the message."
										},
										"uniqueBody": {
											"anyOf": [
												{
													"title": "itemBody",
													"type": "object",
													"properties": {
														"content": {
															"type": "string",
															"description": "The content of the item.",
															"nullable": true
														},
														"contentType": {
															"anyOf": [
																{
																	"title": "bodyType",
																	"enum": [
																		"text",
																		"html"
																	],
																	"type": "string"
																},
																{
																	"type": "object",
																	"nullable": true
																}
															],
															"description": "The type of the content. Possible values are text and html."
														}
													}
												},
												{
													"type": "object",
													"nullable": true
												}
											],
											"description": "The part of the body of the message that is unique to the current message. uniqueBody is not returned by default but can be retrieved for a given message by use of the ?$select=uniqueBody query. It can be in HTML or text format."
										},
										"webLink": {
											"type": "string",
											"description": "The URL to open the message in Outlook on the web.You can append an ispopout argument to the end of the URL to change how the message is displayed. If ispopout is not present or if it is set to 1, then the message is shown in a popout window. If ispopout is set to 0, the browser shows the message in the Outlook on the web review pane.The message opens in the browser if you are signed in to your mailbox via Outlook on the web. You are prompted to sign in if you are not already signed in with the browser.This URL cannot be accessed from within an iFrame.",
											"nullable": true
										},
										"attachments": {
											"type": "array",
											"items": {
												"allOf": [
													{
														"title": "entity",
														"type": "object",
														"properties": {
															"id": {
																"type": "string",
																"description": "The unique identifier for an entity. Read-only."
															}
														}
													},
													{
														"title": "attachment",
														"type": "object",
														"properties": {
															"contentType": {
																"type": "string",
																"description": "The MIME type.",
																"nullable": true
															},
															"isInline": {
																"type": "boolean",
																"description": "true if the attachment is an inline attachment; otherwise, false."
															},
															"lastModifiedDateTime": {
																"pattern": "^[0-9]{4,}-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T([01][0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9]([.][0-9]{1,12})?(Z|[\u002B-][0-9][0-9]:[0-9][0-9])$",
																"type": "string",
																"description": "The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is 2014-01-01T00:00:00Z",
																"format": "date-time",
																"nullable": true
															},
															"name": {
																"type": "string",
																"description": "The attachment\u0027s file name.",
																"nullable": true
															},
															"size": {
																"maximum": 2147483647,
																"minimum": -2147483648,
																"type": "number",
																"description": "The length of the attachment in bytes.",
																"format": "int32"
															}
														}
													}
												]
											},
											"description": "The fileAttachment and itemAttachment attachments for the message."
										},
										"extensions": {
											"type": "array",
											"items": {
												"allOf": [
													{
														"title": "entity",
														"type": "object",
														"properties": {
															"id": {
																"type": "string",
																"description": "The unique identifier for an entity. Read-only."
															}
														}
													},
													{
														"title": "extension",
														"type": "object"
													}
												]
											},
											"description": "The collection of open extensions defined for the message. Nullable."
										},
										"multiValueExtendedProperties": {
											"type": "array",
											"items": {
												"allOf": [
													{
														"title": "entity",
														"type": "object",
														"properties": {
															"id": {
																"type": "string",
																"description": "The unique identifier for an entity. Read-only."
															}
														}
													},
													{
														"title": "multiValueLegacyExtendedProperty",
														"type": "object",
														"properties": {
															"value": {
																"type": "array",
																"items": {
																	"type": "string",
																	"nullable": true
																},
																"description": "A collection of property values."
															}
														}
													}
												]
											},
											"description": "The collection of multi-value extended properties defined for the message. Nullable."
										},
										"singleValueExtendedProperties": {
											"type": "array",
											"items": {
												"allOf": [
													{
														"title": "entity",
														"type": "object",
														"properties": {
															"id": {
																"type": "string",
																"description": "The unique identifier for an entity. Read-only."
															}
														}
													},
													{
														"title": "singleValueLegacyExtendedProperty",
														"type": "object",
														"properties": {
															"value": {
																"type": "string",
																"description": "A property value.",
																"nullable": true
															}
														}
													}
												]
											},
											"description": "The collection of single-value extended properties defined for the message. Nullable."
										}
									}
								}
							]
						}
					}
				}
			}
		]
	},
	"RequestMethod": "Get",
	"RequestUri": "https://graph.microsoft.com:443/v1.0/me/messages?$top=5\u0026$orderby=receivedDateTime\u002Bdesc\u0026$expand=extensions",
	"RequestPayload": null
}
```

Reply:
I've tried playing with the reducer. As far as I understand the interface only provides a way to reduce messages based on the number of messages, not their individual length (please correct me if I missed something).
Even if we could reduce the length of a given message by omitting fields, the biggest offender seems to be the response payload from the first API call (effectively the content property I didn't attach in my previous reply)
Randomly trimming that would lead to invalid JSON, defeating the purpose. And coming up with a solution to randomly prune branches in the JSON tree while keeping the context of the user request, functions etc... might prove difficult.
I'll see if I can "encourage" the LLM to select fewer fields in my instructions.

In the meantime, if there are specific exports you're looking for, please go ahead an provide additional pointers.

Reply:
Hi @baywet, this property was added in scope of this PR - https://github.com/microsoft/semantic-kernel/pull/3450 by Sydney team to validate the function result against the response schema. This property can be useful in scenarios where an LLM needs to reason about an OpenAPI function invocation result as well. 

In cases where it's not needed for certain specific scenarios, it can easily be removed or pruned using [auto function invocation filters]( https://github.com/microsoft/semantic-kernel/blob/3c13912c9da888adffb08fa5fb9efbd47fce6c05/dotnet/samples/Concepts/Filtering/AutoFunctionInvocationFiltering.cs#L141)

Here’s the pseudocode for how it can be implemented:
```csharp
public async Task OnAutoFunctionInvocationAsync(AutoFunctionInvocationContext context, Func<AutoFunctionInvocationContext, Task> next)
{
    await next(context);

    if (context.Result.ValueType == typeof(RestApiOperationResponse))
    {
        var openApiResponse = context.Result.GetValue<RestApiOperationResponse>();
        if (openApiResponse?.ExpectedSchema is not null)
        {
            openApiResponse.ExpectedSchema = null;
        }
    }
}
```
CC: @dmytrostruk

Reply:
This is great guidance and worked right out of the gate! Thank you so much.
For now I have added that to the test itself, since we're still discussing where should those "presets" live In #9841 and #9840